internal/debugger/file_check.go

```diff
@@ -106,11 +106,21 @@ func printConfigFilesTable(w io.Writer, configFiles []ConfigFile) {
 }
 
 func handleFileStatus(file ConfigFile, status FileStatus) {
-	if status == StatusMissing || status == StatusPresentNotReadable || status == StatusNoFile {
+	if status != StatusPresent {
+		message := fmt.Sprintf("%s: %s - %s", getDisplayName(file.Path), file.MissingMsg, file.Purpose)
+
+		// For specific error types, add more context
+		switch status {
+		case StatusMultipleFiles:
+			message = fmt.Sprintf("%s: Multiple configuration files found in directory - %s", getDisplayName(file.Path), file.Purpose)
+		case StatusInvalidJSONFormat:
+			message = fmt.Sprintf("%s: Invalid JSON format - %s", getDisplayName(file.Path), file.Purpose)
+		}
+
 		if file.Required {
-			AddConfigError(fmt.Sprintf("%s: %s - %s", getDisplayName(file.Path), file.MissingMsg, file.Purpose))
+			GetErrorCollector().AddError(message)
```

Comment:
I think you should pass ErrorCollector as a pointer as part of the parameters instead of using GetErrorCollector